src/FilesystemPool.php

```diff
@@ -186,7 +186,7 @@ public function save(CacheItemInterface $item)
 
         $data = serialize($item);
 
-        $result = file_put_contents($this->getPath($key), $data);
+        $result = @file_put_contents($this->getPath($key), $data);
```

Comment:
Please don't use the `@` error suppressor, as it hides all errors it can cause bugs to go unnoticed

Reply:
If we use the ```error suppressor```, it can let ```$result``` be ```false``` and return value.
And it cannot throw the related ```Exception``` directly in executing ```file_put_contents```.

We should figure out another way to handle this. Otherwise the line ```191``` will not reached.

Reply:
I remove the line ```191``` if condition block because this block I think it's not reached.